### DISCUSSION TITLE
自定义主页问题

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1148] [提问] [answered]
联网更新，加载时提示“意外的 DTD 声明”。
如何解决该问题

**Comment by MoYuan-CN** [comment]
附：联网更新地址
https://gitee.com/AlexOng/ccxaml/raw/release_v1.0.0/Custom.xaml
（虽然浏览器打开提示可能包含违规内容，但是PCL2读取没问题？）

**Comment by LTCatt** [comment, marked as answer]
这是 WPF XAML 的提问，和启动器无关，所以不会处理，抱歉（

**Comment by MoYuan-CN** [comment]
然后我昨晚上测试时又好了（雾
